lib/views/invoice_slack_view.ex

```diff
@@ -6,7 +6,7 @@ defmodule Aprb.Views.InvoiceSlackView do
     cond do
       routing_key =~ "merchantaccount" ->
         merchant_account_message(event, partner_data)
-      true ->
+      event["sent_at"] != nil ->
```

Comment:
yes, makes sense, but with current implementation, `verb` is just the current status of the invoice so sent event would still have `unpaid` as `verb` so would use the same `routing_key` but maybe we need to set the verb to `updated` in these cases.